js/helpers.js

```diff
@@ -42,13 +42,28 @@ function generateFilename(prefix, suffix) {
 
     if (CONFIG) {
         if (CONFIG.flightControllerIdentifier) {
-            filename = CONFIG.flightControllerIdentifier + '_' + filename;
-        }
+            filename = CONFIG.flightControllerIdentifier + '_' + CONFIG.flightControllerVersion + "_" + filename;
+        } 
         if (CONFIG.name && CONFIG.name.trim() !== '') {
             filename = filename + '_' + CONFIG.name.trim().replace(' ', '_');
         }
+    } else {
+        // Attempt to get the version number and 
+        MSPHelper.getSetting("version").then(function (data) {
```

Comment:
@MrD-RC are you sure this even works? When CLI is on, MSP on that port no longer works

Reply:
Test 1: Same version
![image](https://user-images.githubusercontent.com/17590174/184678916-06f5d6b9-7511-42d8-8fd9-af06fbbc18ac.png)

Test 2: 4.1.0
![image](https://user-images.githubusercontent.com/17590174/184679876-d724d2bb-7f38-4876-b89e-dbee67ddbe7e.png)

Test 3:
![image](https://user-images.githubusercontent.com/17590174/184680060-d0ef2d71-7586-4e93-8d04-e0266f838a1e.png)

You're right. It's using CONFIG each time. I'll update this as I've seen another improvement.